modules/osdk-golang-run-in-cluster.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-golang-tutorial.adoc
+
+[id="osdk-golang-run-in-cluster_{context}"]
+= Running as a deployment inside the cluster
+
+You can run your Operator project as a deployment inside of your cluster.
+
+.Procedure
+
+. *Build and push the image.*
+
+.. Before building the Operator image, ensure the generated Dockerfile references the base image you want. You can change the default "runner" image `gcr.io/distroless/static:nonroot` by replacing its tag with another, for example `alpine:latest`, and removing the `USER: nonroot:nonroot` directive.
```

Comment:
Should this `gcr.io/distroless/static:nonroot` image be replaced with something more RHEL-relevant for the downstream doc? UBI image?

Reply:
Yes, with `registry.access.redhat.com/ubi8/ubi-minimal:latest`, ex:
```suggestion
.. Before building the Operator image, ensure the generated Dockerfile references a supported base image. You should change the default "runner" image `gcr.io/distroless/static:nonroot` to `registry.access.redhat.com/ubi8/ubi-minimal:latest`, and removing the `USER: nonroot:nonroot` directive.
```

Reply:
Actually documenting substitutions is probably better to do at the [top of the page](https://github.com/openshift/openshift-docs/pull/28372/files#r561284763).

Reply:
@estroz Thanks, I've expanded the substitutions step and create a separate substep for the directive removal. See [updated preview](http://file.rdu.redhat.com/~adellape/010621/osdk_concepts/operators/operator_sdk/golang/osdk-golang-tutorial.html#osdk-golang-run-in-cluster_osdk-golang-tutorial).

> Actually documenting substitutions is probably better to do at the [top of the page](https://github.com/openshift/openshift-docs/pull/28372/files#r561284763).

I couldn't tell what was being linked to here. Where did you think this step should be moved? Under [Creating a new project](http://file.rdu.redhat.com/~adellape/010621/osdk_concepts/operators/operator_sdk/golang/osdk-golang-tutorial.html#osdk-golang-create-project_osdk-golang-tutorial), maybe?

Reply:
My [review](https://github.com/openshift/openshift-docs/pull/28372#pullrequestreview-572628505).